src/core/wb.js

```diff
@@ -255,6 +255,16 @@ var getUrlParts = function( url ) {
 			"#999999"
 		],
 
+		// Get and generate a unique session id
```

Comment:
There is different cache busting strategy that will be available in data-ajax and data-json. The author is free to use the one that best fit his scenario. 

The cache busting for data-ajax plugin is already in a different PR, see #7983